src/translations/strings.yaml

```diff
@@ -1046,18 +1046,21 @@ permissionMacos:
   body:
     value: To use Mozilla VPN, please allow it to run in the background.
     comment: Paragraph requesting the user to allow the VPN to run in the background.
-  instructions:
+  instructionsMacOS14:
     value: In order to do this, please click the button below to open the <i>Login Items</i> of your <i>System Preferences</i>, and enable <i>Allow in the background</i> for Mozilla VPN.
-    comment: Paragraph explaining how to enable the VPN to run in the background.
-  instructionsTahoe:
+    comment: Paragraph explaining how to enable the VPN to run in the background on MacOS 14, see https://support.apple.com/en-by/guide/mac-help/mtusr003/mac for reference translations of these features on different versions of MacOS.
```

Comment:
The name is [macOS](https://www.apple.com/legal/intellectual-property/trademark/appletmlist.html), not MacOS. 

Change links to https://support.apple.com/guide/mac-help/mtusr003/mac (drop the `en-by`, which is English in Belarus of all places). That will redirect the user to the best locale.